templates/patterns/popover/popover.ui_patterns.yml

```diff
@@ -0,0 +1,63 @@
+popover:
+  label: "Popover"
+  description: "The Popover JavaScript plugin is used to show and hide content. Buttons are used as triggers that are mapped to specific elements you toggle. https://v5.getbootstrap.com/docs/5.0/components/popover/"
+  variants:
+    default:
+      label: Button
+      description: Show popover in a button.
+    link:
+      label: Link
+      description: Show popover in a link.
+  settings:
+    variant:
+      type: select
+      label: Button or link style
+      options:
+        primary: primary
+        secondary: secondary
+        success: success
+        danger: danger
+        warning: warning
+        info: info
+        light: light
+        dark: dark
+        link: link
+      preview: link
+    outline:
+      type: boolean
+      label: Button outline
+    size:
+      type: select
+      label: Button size
+      options:
+        sm: small
+        lg: large
+    placement:
+      type: select
+      label: Popover placement
+      options:
+        top: top
+        right: right
+        bottom: bottom
+        left: left
+      preview: top
+  fields:
+    label:
+      type: "text"
+      label: "Label"
+      description: "Button or link label"
+      preview: "Toggle element"
+    title:
+      type: "render"
+      label: "Title"
+      description: "Popover title"
+      preview: "Lorem ipsum dolor sit amet"
+    content:
+      type: "render"
+      label: "Content"
+      description: "Popover content"
+      preview: "Lorem ipsum dolor sit amet, consectetur adipiscing elit."
```

Comment:
add a new field, can be boolean "is_link" or can be select "trigger_type"

Reply:
I realized this is not needed. Checking bootstrap page (https://getbootstrap.com/docs/5.0/components/popovers/) and netlify (https://oelibrary.netlify.app/default/?path=/story/components-button--popover&args=icon_size:!undefined;variant:link;name:!undefined;transformation:!undefined), this is not needed at all because is all about button style.